feat: Implement twist_controller

JJMats opened the pull request:
Added initial code to twist_controller to create the Controller
object and subscribed to necessary topics.

Review ros/src/twist_controller/explanation.txt for details

JJMats committed:
feat: Implement twist_controller
Added initial code to twist_controller to create the Controller
object and subscribed to necessary topics.

JJMats committed:
style: Add comments and remove extraneous code
Removed semi-colon from yaw_controller.py, resolved typos, and
added extra logwarn messages for debugging.

emoyers commented:
Hi  guys,
May I merge Jeremy's Pull request to
https://github.com/truongconghiep/CarND-Capstone?
I guess this will be the repo we are delivering, right?

Best regards,
Emilio Moyers

On Tue, Apr 16, 2019 at 3:19 AM Jeremy Matson <notifications@github.com>
wrote:

> @JJMats <https://github.com/JJMats> requested your review on: #3
> <https://github.com/truongconghiep/CarND-Capstone/pull/3> feat: Implement
> twist_controller.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/truongconghiep/CarND-Capstone/pull/3#event-2278727295>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APA4XpuNs_COTNcLB5zHf7ZtHr96H0qqks5vhXmCgaJpZM4cwg5r>
> .
>

JJMats committed:
feat: Improve brake torque calculation
Added the mass of the vehicle's fuel load to the vehicle_mass for
calculating braking torque in twist_controller.py.
